web-ng/src/app/components/share-dialog/share-dialog.component.ts

```diff
@@ -56,52 +77,107 @@ export class ShareDialogComponent {
     this.subscription.add(
       this.projectService
         .getActiveProject$()
+        .pipe(take(1))
```

Comment:
just to educate me.. why did you add it?:P

Reply:
Added a clarifying comment. HTH!

Reply:
nice!